Packs/RunZero/Integrations/RunZero/RunZero.py

```diff
@@ -0,0 +1,525 @@
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        client.quota_get()
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def wireless_lan_search(client: Client, args: dict) -> CommandResults:
+    wireless_string = ''
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    if args.get('wireless_id'):
+        wireless_string = f'/{args["wireless_id"]}'
+    elif args.get('search'):
+        wireless_string = f'?search={args["search"]}'
+    raw = client.wireless_search(wireless_string)
+    raw = raw[:limit]
+    message = []
+    if type(raw) is list:
+        raw = raw[:limit]
+        for item_raw in raw:
+            message.extend(parse_raw_wireless(item_raw))
+    if type(raw) is dict:
+        message.extend(parse_raw_wireless(raw))
+    human_readable = tableToMarkdown('Wireless',
+                                     message,
+                                     removeNull=True)
+    return CommandResults(
+        outputs_prefix='RunZero.WirelessLAN',
+        outputs_key_field='id',
+        outputs=raw,
+        raw_response=raw,
+        readable_output=human_readable
+    )
+
+
+def wireless_lan_delete(client: Client, args: dict) -> CommandResults:
+    wireless_id = args.get('wireless_id', '')
+    raw = client.wireless_delete(wireless_id)
+    message = f'Wireless LAN {wireless_id} deleted successfully.'
+    return CommandResults(
+        outputs_prefix=None,
+        outputs_key_field=None,
+        outputs=None,
+        raw_response=raw,
+        readable_output=message
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+    :return:
+    :rtype:
+    """
+    params = demisto.params()
+    api_key = params.get('api_key', {}).get('password')
+    base_url = urljoin(params.get('url'), '/api/v1.0')
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        headers = {'Authorization': f'Bearer {api_key}'}
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers=headers,
+            proxy=proxy)
+
+        args = demisto.args()
+        if demisto.command() == 'test-module':
+            result = test_module(client)
+            return_results(result)
```

Comment:
just wrap the result with `return_results()` - no need for it to be a 2 liner.
The same goes for all the rest of the commands.
```suggestion
            return_results(test_module(client))
```